pkg/client/kubernetes/types.go

```diff
@@ -77,6 +78,7 @@ func init() {
 		resourcesscheme.AddToScheme,
 		hvpav1alpha1.AddToScheme,
 		druidv1alpha1.AddToScheme,
+		apiextensionsv1.AddToScheme,
```

Comment:
Wouldn't
```suggestion
		apiextensionsscheme.AddToScheme,
```
be better?

Reply:
It's better to use the scheme from the versioned resource

Reply:
Why? The `apiextensionsscheme.AddToScheme` adds all versions to the scheme and not only a concrete one. This might be needed/helpful as Gardener is working with Kubernetes clusters of several versions.